spec/routes/redirections_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+require "decidim/dev/test/spec_helper"
+require "decidim/core/test/factories"
+require "decidim/proposals/test/factories"
+
+describe "routing redirections", type: :request do
```

Comment:
Actually I don't think so, it's a routing request *hacked* so I can test what I want. Moving it to `requests` might be misleading.